analysis/src/HistoManager.cxx

```diff
@@ -104,11 +110,14 @@ void HistoManager::DefineHistos(std::vector<std::string> histoCopyNames, std::st
                 h_name = m_name+"_"+ histoCopyNames.at(i) + "_" + hist.key() ;
                 singleCopy = false;
             }
-
-            std::cout << "DefineHisto: " << h_name << std::endl;
-            std::size_t found = (hist.key()).find_last_of("_");
-            std::string extension = hist.key().substr(found+1);
-
+            if(debug_){
```

Comment:
We might as well just have one if(debug_) in this section of the code, and just cout all the stuff in that one, rather than the two.

Reply:
Done.